content/ember/v5/deprecate-template-action.md

```diff
@@ -0,0 +1,170 @@
+---
+title: Action helper and modifier 
+until: 6.0.0
+since: 5.9.0
+---
+
+### Scenario: `action` is passed a string
+
+Before:
+```handlebars
+<button type="button" {{action "plusOne"}}>
+  Click Me
+</button>
+```
+
+After
+
+```handlebars
+<button type="button" {{on 'click' this.plusOne}}>
+  Click Me
+</button>
+```
+or, if `plusOne` is passed in as an argument 
+```handlebars
+<button type="button" {{on 'click' @plusOne}}>
+  Click Me
+</button>
+```
+
+If the `plusOne` action is in an actions object, it needs to move out:
+
+Before:
+```javascript
+import Component from '@glimmer/component';
+
+export default class Demo extends Component {
+    actions = {
+        plusOne() {
+           /* ... */ 
+        }
+    }
+}
+```
+After:
+```javascript
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
+
+export default class Demo extends Component {
+    @action
+    plusOne() {
+       /* ... */ 
+    }
+}
+```
+
+or
+
+Before:
+```javascript
+import Component from '@ember/component';
+
+export default class Demo extends Component {
+    actions = {
+        plusOne() {
+           /* ... */ 
+        }
+    }
+}
+```
+
+After:
+```javascript
+import Component from '@ember/component';
+import { action } from '@ember/object';
+
+export default class Demo extends Component {
+    @action
+    plusOne() {
+       /* ... */ 
+    }
+}
+```
+or
+
+Before:
+```javascript
+import Component from '@ember/component';
+
+export default Component.extend({
+    actions: {
+        plusOne() {
+           /* ... */ 
+        }
+    }
+})
+```
+
+After:
+```javascript
+import Component from '@ember/component';
+import { action } from '@ember/object';
+
+export default Component.extend({
+  plusOne: action(function() {
+      /* ... */ 
+  }),
+})
```

Comment:
We should also capture the fact someone might be calling `this.send('plusOne');` which needs to change to `this.plusOne()`;